### PR TITLE
added support for inputing mood yesterday

### PR DESCRIPTION
**Pull request by raynald**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ rtm.on(CLIENT_EVENTS.RTM.RTM_CONNECTION_OPENED, () => {
 const A_WEEK_IN_SECONDS = 604800;
 
 function isEmoji(str) {
-  return /^:[\w-]+:$/.test(str);
+  return /^:[\w-+]+:$/.test(str);
```
Comment (dangrasso) [inline src/index.js:28]: why? Is this related to this feature?

Comment (raynald) [reply src/index.js:28]: Because, we have emoji 👍  with code (:"+1":)

File: src/index.js [hunk 2]
```diff
@@ -240,16 +240,20 @@ rtm.on(RTM_EVENTS.MESSAGE, (message) => {
     }
   } else if (message.text.toLowerCase() === 'help') {
     rtm.sendMessage('*command list*\n' +
-        '> feel [emoji] [1-6] ([snippet]) `tell the bot how you feel now, snippet is optional`\n' +
+        '> feel/felt [emoji] [1-6] ([snippet]) `tell the bot how you feel/felt now, snippet is optional`\n' +
```
Comment (dangrasso) [inline src/index.js]: > ...how you feel/felt now...

This is a bit confusing. What about showing them as separate commands in the help instead?

Comment (raynald) [reply src/index.js]: yes! good suggestion.

File: src/index.js [hunk 3]
```diff
@@ -240,16 +240,20 @@ rtm.on(RTM_EVENTS.MESSAGE, (message) => {
...
         '```1 (depressed), 2 (sad), 3 (unhappy), 4 (satisfied), 5 (joyful), 6 (exuberant)```'
       , message.channel);
-  } else if (message.text.substr(0, 4).toLowerCase() === 'feel') {
-    const timestamp = getTimestampInSeconds(message);
+  } else if (message.text.substr(0, 4).toLowerCase() === 'feel' || message.text.substr(0, 4).toLowerCase() === 'felt') {
+    const command = message.text.substr(0, 4).toLowerCase();
+    let timestamp = getTimestampInSeconds(message);
+    if (command === 'felt') {
+      timestamp -= 86400;
+    }
```
Comment (dangrasso) [inline src/index.js]: This is a trick to handle two separate commands in the same else if branch.
What about treating `feel` and `felt` as separate commands? 
This would simplify a bit the implementation of each one. We could then extract the common parts in functions.

Comment (raynald) [reply src/index.js]: I tried to reuse the code, because these two commands are more-or-less the same

**Review by dangrasso — COMMENTED** [review]
This is a cool feature, but I think we should consider the `felt` command as a separate one instead of forcing it into the `feel` implementation.

This also because we should accept a date argument in the `felt` case. Even if we don't add that functionality in this PR, we know it's coming :)

File: src/index.js (hunk 1 above)
Comment (dangrasso) [reply src/index.js:28]: ok, so it is unrelated ;)

**Comment by raynald** [comment]
Adding a date feature is tricky though. 😺  We should discuss how we are gonna do it.

**Review by dangrasso — APPROVED** [review]
